readers/mongodb/messages_test.go

```diff
@@ -56,31 +51,55 @@ func TestReadSenml(t *testing.T) {
 	db := client.Database(testDB)
 	writer := writer.New(db)
 
+	chanID, err := uuid.New().ID()
+	require.Nil(t, err, fmt.Sprintf("got unexpected error: %s", err))
+	pubID, err := uuid.New().ID()
+	require.Nil(t, err, fmt.Sprintf("got unexpected error: %s", err))
+	pub2ID, err := uuid.New().ID()
```

Comment:
Why not `pubID2`? It's a second ID.

Reply:
It's also a second publisher. ID of publisher2

Reply:
I would also prefer `pubID2`, because this look like `pub-to-ID`. In functional languages we often see these vars with numbers in the end (because vars are immutable, and you have to add new with the practically same name).